webservices/common/models/dates.py

```diff
@@ -85,7 +85,7 @@ class ElectionClassDate(db.Model):
     senate_class = db.Column(db.Integer, index=True, doc=docs.SENATE_CLASS)
 
 
-class CalendarDate(BaseModel):
+class CalendarDate(db.Model):
```

Comment:
We already have idx in the calendar date model as event_id. For the calendar table we needed a id so you could pull up a single event. This is where the warning about ids on  calendar date comes from when you run ./manage.py